Training/TPU-Trillium/Diffusion-2-PyTorch/README.md

```diff
@@ -0,0 +1,43 @@
+# Instructions for training Stable Diffusion 2 on TPU Trillium
+
+
+This user guide provides a concise overview of the essential steps required to run StableDiffusion 2.0 base training on Cloud TPUs.
+
+
+## Environment Setup
+
+The following setup assumes to run the training job with StableDiffusion 2.0 base on GCE TPUs using the docker image from this registery (us-central1-docker.pkg.dev/tpu-pytorch/docker/development/pytorch-tpu-diffusers:v2), the docker image uses the pytorch and torch_xla nightly build from 09/05 and has all the package dependency installed. It cloned the git repo from [https://github.com/pytorch-tpu/diffusers (commit f08dc9)](https://github.com/pytorch-tpu/diffusers/tree/f08dc92db9d7fd7d8d8ad4efcdfee675e2cd26f2) in order to run hugging face stable diffusion on TPU. Please follow corresponding TPU generation's user guide to setup the GCE TPUs first. All the command below should run from your own machine (not the TPU host you created).
```

Comment:
Do you want to upload to `deeplearning-images/us-central1/reproducibility` instead of `tpu-pytorch/docker/development`? The `deeplearning-images` registry is actually public.

Reply:
Good to know! Uploaded to us-central1-docker.pkg.dev/deeplearning-images/reproducibility/pytorch-tpu-diffusers:v4 for public sharing.